xrpl/utils/tx_parser.py

```diff
@@ -0,0 +1,197 @@
+from xrpl.utils.xrp_conversions import drops_to_xrp
+import pydash
+
+"""
+This is a rewritten repository of https://github.com/ripple/ripple-lib-extensions/blob/master/transactionparser/src/balancechanges.js in python.
+"""
+
+### Utils ###
+class Normalized_Node():
```

Comment:
Class names shouldn't have underscores in them
```suggestion
class NormalizedNode():
```